.github/workflows/centos7.yml

```diff
@@ -41,45 +70,47 @@ jobs:
         buildargs: ANTARES_DEPS_VERSION
 
   build:
-    name: Build (XPRESS + OR-Tools=${{ matrix.ortools }})
+    name: Build (XPRESS + OR-Tools=${{ matrix.ortools-xpress }})
     env:
       XPRESSDIR: ${{ github.workspace }}/xpressmp813
       ORTOOLSDIR: ${{ github.workspace }}/or-tools
 
     runs-on: ubuntu-latest
-    needs: docker_publish
+    needs:
+      - docker_publish
+      - doc_generation
     container: 'antaresrte/rte-antares:centos7-simulator'
     strategy:
       matrix:
-        ortools: [ON, OFF]
+        ortools-xpress: [ON, OFF]
 
     steps:
-           
     - uses: nelonoel/branch-name@v1.0.1
-    
+
     - name: Checkout
       run: |
           git clone $GITHUB_SERVER_URL/$GITHUB_REPOSITORY.git -b ${BRANCH_NAME} .
-    
+
     - name: Get release
       id: get_release
       uses: bruceadams/get-release@v1.2.0
-           
+      if: ${{ env.IS_RELEASE == 'true' }}
+
     - name: Init submodule
       run: |
-           git submodule update --init --recursive src
-           
+          git submodule update --init --recursive src/antares-deps src/tests/resources/Antares_Simulator_Tests
+
     - name: Download XPRESS if needed
-      if: ${{ matrix.ortools == 'ON' }}
+      if: ${{ matrix.ortools-xpress == 'ON' }}
       run: |
         source /opt/rh/rh-git227/enable
         git clone -b master https://${{ secrets.XPRS_USERNAME }}:${{ secrets.XPRS_TOKEN }}@github.com/rte-france/xpress-mp ${{ env.XPRESSDIR }}
 
     - name: Download OR-Tools if needed
-      if: ${{ matrix.ortools == 'ON' }}
+      if: ${{ matrix.ortools-xpress == 'ON' }}
       id: ortools
       run: |
-          if [ "${{ matrix.ortools }}" == "ON" ]; then
+          if [ "${{ matrix.ortools-xpress }}" == "ON" ]; then
             mkdir -p ${{ env.ORTOOLSDIR }} && cd ${{ env.ORTOOLSDIR }}
             wget -q -O ortools.zip https://github.com/rte-france/or-tools/releases/download/v9.2-rte1.2/ortools_cxx_centos7_shared_xprs.zip
```

Comment:
Download pre-compiled OR-Tools in all cases, with a different URL (same as Ubuntu). Make sure to use `static` assets.